src/ui/flutter_app/lib/shared/sized_spacer.dart

```diff
@@ -0,0 +1,29 @@
+/*
+  This file is part of Sanmill.
+  Copyright (C) 2019-2021 The Sanmill developers (see AUTHORS file)
+
+  Sanmill is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Sanmill is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import 'package:flutter/material.dart';
+import 'package:sanmill/shared/theme/app_theme.dart';
+
+class CustomSpacer extends StatelessWidget {
+  const CustomSpacer({Key? key}) : super(key: key);
```

Comment:
Or, change the class name to SizedSpacer?

Reply:
done